src/application/__init__.py

```diff
@@ -0,0 +1,6 @@
+from constants import NOTES_DIR
+
+
+def display_path(path: str) -> str:
+    """Replaces NOTES_DIR with 'Explorer'"""
+    return path.replace(NOTES_DIR, "Explorer")
```

Comment:
Nice change! Does this belong in `__init__.py` this is kind of setting off style alarm bells for me, but I'm not sure.